src/gpuarray_buffer_opencl.c

```diff
@@ -1152,6 +1152,14 @@ static int cl_property(gpucontext *c, gpudata *buf, gpukernel *k, int prop_id,
     size_t *psz;
     cl_device_id id;
     cl_uint ui;
+    /* For GA_CTX_PROP_PCIBUSID (currently desactivated).
+     * According to http://www.makelinux.net/ldd3/chp-12-sect-1
+     * (accessed on 2016/11/09:15h41 EST):
+     * domain: 16 bits, bus: 8 bits, device: 5 bits, function: 3 bits. */
+    /*
+    uint32_t* busid;
+    uint32_t domain = 0, bus = 0, device = 0, function = 0;
+    */
```

Comment:
This can help for the OpenCL side: https://anteru.net/blog/2014/08/01/2483/

But otherwise, DEVSUP is fine.